src/hooks/useTypeaheadFocus.ts

```diff
@@ -4,12 +4,20 @@ import {useProvidedRefOrCreate} from './useProvidedRefOrCreate'
 
 export const TYPEAHEAD_TIMEOUT = 1000
 
-export const useTypeaheadFocus = (open: boolean, providedRef?: React.RefObject<HTMLElement>) => {
-  const containerRef = useProvidedRefOrCreate(providedRef)
+export const useTypeaheadFocus = (
+  open: boolean,
+  providedContainerRef: React.RefObject<HTMLElement>,
+  providedAnchorRef?: React.RefObject<HTMLElement>
```

Comment:
comment: typeahead should activate when the menu is opened with keyboard and mouse both.

https://user-images.githubusercontent.com/1863771/159458072-396e6226-9d05-4e53-abde-b9b486be053c.mov